src/App.js

```diff
@@ -1,13 +1,27 @@
 import './App.css';
-import Navbar from './components/Navbar/Navbar';
-import logo from './logo.png';
+
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Login from './pages/Login/Login';
-import { Link } from 'react-router-dom';
+import Home from './pages/Home/Home';
+import Signup from './pages/Signup/Singup';
+import WishList from './pages/WishList/WishList';
+import SingleProductPage from './pages/SingleProductPage/SingleProductPage';
+import Product from './pages/Product/Product';
+import Cart from './pages/Cart/Cart';
```

Comment:
As these all are coming from same directory of pages, so you could create an `index.js` file in your pages folder and export all these pages from that file and import them here all together.
e.g.
```jsx
import {Home, Login, Signup, etc} from "./pages";
```

This helps in reducing the no. of lines as well as bundle all the imports together of on directory

Reply:
I will do the refactor vishal
Thank you for the feedback